hopsworksai/resource_cluster.go

```diff
@@ -685,8 +685,7 @@ func awsAttributesSchema() *schema.Resource {
 						"subnet_id": {
 							Description: "The subnet id.",
 							Type:        schema.TypeString,
-							Optional:    true,
-							Computed:    true,
+							Required:    true,
```

Comment:
Won't this make that we can't create a cluster without creating a subnet first?

Reply:
only if you add a network block, you will need to define the vpcId and the subnetId